iree/compiler/Conversion/HLOToLinalg/HLOToLinalgOnBuffers.cpp

```diff
@@ -591,33 +591,60 @@ LogicalResult ReduceWindowOpConversion::apply(
     mhlo::ReduceWindowOp op, ArrayRef<Value> inputBuffers,
     ArrayRef<Value> resultBuffers, ConversionPatternRewriter &rewriter) const {
   auto loc = op.getLoc();
+  auto resultType = op.getResult().getType().cast<ShapedType>();
+  int rank = resultType.getRank();
```

Comment:
Super nit: We can do `if (resultType.getRank() != 4)` given `rank` is not used afterwards.